packages/hw-app-eth/src/Eth.js

```diff
@@ -240,6 +249,22 @@ export default class Eth {
       toSend.push(buffer);
       offset += chunkSize;
     }
+
+    let decodedTx = decodeTx("0x" + rawTxHex);
+    if (decodedTx.data.length >= 10) {
+      const erc20Info = byContractAddress(decodedTx.to);
+      console.log(erc20Info);
+      if (erc20Info) {
+        this._provideERC20TokenInformation(erc20Info);
```

Comment:
in live-common, we also have to do this, specifically for compound to also provide the c* token data
https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/ethereum/signOperation.js#L72-L78

how do we plan to address this? would it still be needed?

Reply:
the approve function selector is 0x095ea7b3
https://www.4byte.directory/signatures/?bytes4_signature=0x095ea7b3

we are going to update coumpound b2c json file to add this selector and set the erc20OfInterest key.
See paraswap https://github.com/LedgerHQ/crypto-assets/pull/204/files#diff-f1d134479e4f1cfc1ffe64248098a166ffe57fbc320bfb8dffdad190d10d2b70

Reply:
actually after discussion with @TamtamHero
- its USDT.approve(cUSDC) that is actually made as call.
- Instead of checking the ticker in the device app to know if its a coumpound approval they will hardcode directly the address

=> no more need to call proviceERC20TokenInformation

Reply:
=> we need to leave the extra call in live common to ensure compatibility with previous ethereum app version

Reply:
https://github.com/LedgerHQ/app-ethereum/commit/43e7badfa4e0b4463f12483f556a048d6177b29f